src/utils/encryption.js

```diff
@@ -1,9 +1,8 @@
 import { sha3_256 } from "js-sha3";
 import util from "node-forge/lib/util";
 import random from "node-forge/lib/random";
-import md from "node-forge/lib/md";
+import md from "node-forge/lib/sha512";
```

Comment:
Will this md give us access to all the other forge hashing methods, or just sha512 and sha384?  If it will only give us access to those, can we alias it differently than just "md"?

Reply:
Actually we should probably just call it "sha512" since there's a different thing from node-forge called "md" that we could import.

Reply:
this is just importing the sha512 file

Reply:
i can rename it to SHA512 since that is the file we are importing

Reply:
I understand that, but there's a totally different thing in node-forge called "md" which we might also need to import at some point.  I don't think we should take that name.

Reply:
Sounds good.